src/services/firebase/firestore.ts

```diff
@@ -196,7 +216,11 @@ export class FirestoreServices {
 
       const messageRef = firestore()
         .collection<SendMessageProps>(
-          `${FireStoreCollection.conversations}/${this.conversationId}/${FireStoreCollection.messages}`
+          `${
```

Comment:
Do you think we need to create a function to get `conversation` and `user` path? I see we can reuse it many times in this file.

Reply:
Updated